paasta_tools/paasta_metastatus.py

```diff
@@ -128,7 +184,7 @@ def main(argv=None):
     except MasterNotAvailableException as e:
         # if we can't connect to master at all,
         # then bomb out early
-        paasta_print(PaastaColors.red("CRITICAL:  %s" % e.message))
+        paasta_print(PaastaColors.red("CRITICAL:  %s" % '\n'.join(e.args)))
```

Comment:
also to make mypy happy -- BaseException doesn't have a .message attribute in py3: https://stackoverflow.com/a/13063924